doc/sphinx-guides/source/style/text.rst

```diff
@@ -0,0 +1,14 @@
+Text
+++++
+
+Here we describe the guidelines that help us provide helpful, clear and consistent textual information to users.
+
+.. contents:: |toctitle|
+  :local:
+
+Metadata Text Guidelines
+========================
+
+`Bootstrap <http://getbootstrap.com/css/#grid>`__ provides a responsive, fluid, 12-column grid system that we use to organize our page layouts.
+
```

Comment:
```suggestion
```

This line about Bootstrap should be deleted, right? It seems be be copied from the "foundations" page.

Reply:
Yeah, I agree. Not sure how that line got on this Text page. Thanks for catching that!